roles/openshift_logging/tasks/start_cluster.yaml

```diff
@@ -19,107 +22,125 @@
   loop_control:
     loop_var: fluentd_host
 
-- command: >
-    {{openshift.common.client_binary}} --config={{mktemp.stdout}}/admin.kubeconfig get dc -l component=es -o name -n {{openshift_logging_namespace}}
+- name: Retrieve elasticsearch
+  oc_obj:
+    state: list
+    kind: dc
+    selector: "component=es"
+    namespace: "{{openshift_logging_namespace}}"
+    kubeconfig: "{{mktemp.stdout}}/admin.kubeconfig"
   register: es_dc
-  check_mode: no
-  changed_when: no
 
 - name: start elasticsearch
   oc_scale:
     kind: dc
-    name: "{{object.split('/')[1]}}"
+    name: "{{ object }}"
     namespace: "{{openshift_logging_namespace}}"
     kubeconfig: "{{mktemp.stdout}}/admin.kubeconfig"
     replicas: 1
-  with_items: "{{es_dc.stdout_lines}}"
+  with_items: "{{ es_dc.results.results[0]['items'] | map(attribute='metadata') | map(attribute='name') | list }}"
```

Comment:
I wish there was a better way to do it, e.g. ``results.results[0].items | map(attribute='metadata.name')``